### DISCUSSION TITLE
Ability to hide all menus

### DISCUSSION THREAD

**Discussion by ChaosWebDev** [NativePHP/laravel#415] [Ideas]
Can we add the ability to hide all menus (or remove them)?

**Comment by ChaosWebDev** [comment]
On windows, in the `NativeAppServiceProvider.php` file, you can do `Window::open()->hideMenu();` and it seems to work.

An alternate method that worked for me is to do 2 lines:

`Menu::new()->register();`
`Window::open();`

**Reply by simonhamp** [reply]
[Docs](https://nativephp.com/docs/1/the-basics/windows#hiding-the-menu)
